### ISSUE TITLE
Plotting small meshes `plot`

### ISSUE THREAD

**Issue by krober10nd** [CHLNDDEV/OceanMesh2D#126]
* Often times small meshes are not immediately visible when plotting (i.e., `plot`). 
* The user has to scroll around randomly to find it on the screen. 
* Is this related to the axis sizing in `simpplot`?

**Comment by WPringle** [comment]
yes it must be related to the way `simpplot` is handling things. It works fine when plotting with the projection.
